nim/chain.json

```diff
@@ -0,0 +1,83 @@
+{
+    "$schema": "../chain.schema.json",
```

Comment:
you use a mix of tabs sometimes being 4 spaces and sometimes being 2 spaces. Can we stick to use 2 spaces everywhere?